rocky/rocky/settings.py

```diff
@@ -343,24 +343,24 @@
 TAGULOUS_SLUG_ALLOW_UNICODE = True
 
 TAG_COLORS = [
-    ("blue-light", _("Blue light")),
-    ("blue-medium", _("Blue medium")),
-    ("blue-dark", _("Blue dark")),
-    ("green-light", _("Green light")),
-    ("green-medium", _("Green medium")),
-    ("green-dark", _("Green dark")),
-    ("yellow-light", _("Yellow light")),
-    ("yellow-medium", _("Yellow medium")),
-    ("yellow-dark", _("Yellow dark")),
-    ("orange-light", _("Orange light")),
-    ("orange-medium", _("Orange medium")),
-    ("orange-dark", _("Orange dark")),
-    ("red-light", _("Red light")),
-    ("red-medium", _("Red medium")),
-    ("red-dark", _("Red dark")),
-    ("violet-light", _("Violet light")),
-    ("violet-medium", _("Violet medium")),
-    ("violet-dark", _("Violet dark")),
+    ("color-1-light", _("Blue light")),
+    ("color-1-medium", _("Blue medium")),
+    ("color-1-dark", _("Blue dark")),
+    ("color-2-light", _("Green light")),
+    ("color-2-medium", _("Green medium")),
+    ("color-2-dark", _("Green dark")),
+    ("color-3-light", _("Yellow light")),
+    ("color-3-medium", _("Yellow medium")),
+    ("color-3-dark", _("Yellow dark")),
+    ("color-4-light", _("Orange light")),
+    ("color-4-medium", _("Orange medium")),
+    ("color-4-dark", _("Orange dark")),
+    ("color-5-light", _("Red light")),
+    ("color-5-medium", _("Red medium")),
+    ("color-5-dark", _("Red dark")),
+    ("color-6-light", _("Violet light")),
+    ("color-6-medium", _("Violet medium")),
+    ("color-6-dark", _("Violet dark")),
```

Comment:
I don't see any mention why this needs to be changed. Also these are values that are saved in the database, so we would have to do a data migration if we change this. But then I don't really see what the point of doing all that work to switch to color-X if we specify the color name in the description anyway?

Reply:
Good point about the migration thing, I wasn't aware of that. At first, I thought the naming had something to do with Manon.

Reply:
@dekkers Yes, these values have changed in the new Manon. I will add the migration file! thanks for putting this out.